contracts/eosio.tonomy/src/eosio.tonomy.cpp

```diff
@@ -140,20 +140,29 @@ namespace eosiotonomy
 
    void bios::onblock(ignore<block_header> header)
    {
-      int64_t current_time = eosio::current_time_point().time_since_epoch().count();
+      eosio::time_point now = eosio::current_time_point();
+      eosio::print("{\"event_log\":{\"account\":\"eosio\",\"action\":\"onblock\"},\"time\":\"", now.to_string(), "Z\",\"events\":[");
+      int64_t current_time = now.time_since_epoch().count();
       int64_t time_rounded_to_half_periods = (current_time + (half_cron_period/2)) / half_cron_period * half_cron_period; // Round to the nearest half-second
 
+      bool first_event = true;
+
       // at half past each cron period, call the staking.tmy::cron() action
       // FIXME: this is calling on every block still (I think), not every cron period
```

Comment:
the logging shows that this is indeed true. we should fix this before launching staging.